specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -2200,6 +2200,7 @@
         },
         "lifetimeActions": {
           "type": "array",
+          "required": true,
```

Comment:
This is an SDK-breaking change. You can make something required to be not-required, but not this way.

Reply:
...besides, `required` is an array property on a model object. It can only be authored as a boolean property for *parameters*.

See:
* https://swagger.io/specification/#parameter-object
* https://swagger.io/specification/#schema-object